src/App.jsx

```diff
@@ -2,7 +2,7 @@ import { Routes, Route } from "react-router-dom";
 
 import { Home, Registration } from "./Pages";
 
-import { Navbar, Footer } from "./Components";
+import { Navbar, Footer, Sponsors } from "./Components";
```

Comment:
Move Sponsors from `Components` directory to `Pages` directory as, that's a new page apart from Home Page, 

and then create a new route after importing the Sponsor from Pages as 

```jsx
import {Home, Registration, Sponsors} from "./Pages"
```

```jsx
<Route path="/sponsors" element={<Sponsors/>}/>
```